api/src/main/java/com/minirogue/api/media/MediaEnums.kt

```diff
@@ -47,9 +47,9 @@ enum class MediaType(val legacyId: Int) {
     @SerialName("Audiobook")
     AUDIOBOOK(14);
 
-    fun getSerialname() = MediaType.serializer().descriptor.getElementName(ordinal)
+    fun getSerialName() = MediaType.serializer().descriptor.getElementName(ordinal)
```

Comment:
camelCase nit while I was working in this file (causes several other diffs in this PR)